### ISSUE TITLE
Also affects ASUS-WL500g

### ISSUE THREAD

**Issue by awalls-cx18** [jduck/asus-cmd#1]
This issue also affects the ASUS WL-500g (from about 9 years ago).  The format of the infosrvr protocol header is probably different.  Here was some old code i had to exercise one of the innocuos OpCodes:

https://github.com/awalls-cx18/wl500g-infoclient/blob/master/infoclient.c#L55

**Comment by jduck** [comment]
Interesting! Was this reported back then?

**Comment by awalls-cx18** [comment]
No.  I never thought it was that big a deal (in both 2005 or 2013) since it was a local network exploit on a router intended for home use.

Another interesting bit was documented here on a really old firmware version:
http://oleg.wl500g.info/
"Removed never ending ping done by firmware, notifying guys from Taiwan of your router, when you press Connect"

I never found any evidence of that in my firmware images, but I didn't look incredibly hard.

**Comment by jduck** [comment]
If you have to start it manually I'm not sure we can call  that vulnerable.
